src/Template/Aefis/xml/e2b.ctp

```diff
@@ -140,10 +144,10 @@
             <receiveremailaddress/>
         </receiver>
         <patient>
-            <patientinitial><?php echo $aefi['patient_name']; ?></patientinitial>
-            <patientgpmedicalrecordnumb><?php echo $aefi['patient_next_of_kin']; ?></patientgpmedicalrecordnumb>
-            <patientspecialistrecordnumb><?php echo $aefi['patient_next_of_kin']; ?></patientspecialistrecordnumb>
-            <patienthospitalrecordnumb><?php echo $aefi['patient_next_of_kin']; ?></patienthospitalrecordnumb>
+            <patientinitial><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$aefi['patient_name']); ?></patientinitial>
+            <patientgpmedicalrecordnumb><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$aefi['patient_next_of_kin']); ?></patientgpmedicalrecordnumb>
+            <patientspecialistrecordnumb><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$aefi['patient_next_of_kin']); ?></patientspecialistrecordnumb>
+            <patienthospitalrecordnumb><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$aefi['patient_next_of_kin']); ?></patienthospitalrecordnumb>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate